modules/ingester/instance.go

```diff
@@ -337,7 +337,8 @@ func (i *instance) writeTraceToHeadBlock(id common.ID, b []byte) error {
 }
 
 func (i *instance) Combine(objA []byte, objB []byte) []byte {
-	return util.CombineTraces(objA, objB)
+	combinedTrace, _ := util.CombineTraces(objA, objB)
```

Comment:
log the error if it's set

Reply:
https://github.com/grafana/tempo/pull/480/files#r565891495